### PR TITLE
Feature / dqa updates

### PR DESCRIPTION
**Pull request by alliuca**
**Description:** Contains various DQA updates.

**Ticket:** https://theeventscalendar.atlassian.net/browse/TEC-3766

**Screenshots:**

Screen Shot 2021-03-04 at 6 35 20 PM

Screen Shot 2021-03-04 at 6 35 28 PM

Screen Shot 2021-03-04 at 6 35 35 PM

Screen Shot 2021-03-04 at 6 38 11 PM

### CODE REVIEW

File: src/resources/postcss/views/full/_single-event.pcss [hunk 1]
```diff
@@ -231,11 +258,10 @@
 .tribe-events-meta-group {
 
 	.tribe-events-single-section-title {
-		@mixin heading;
-		@mixin heading-8;
-
-		color: var(--color-text-primary-light);
-		font-weight: var(--font-weight-regular);
+		color: var(--color-text-primary);
+		font-size: 15px;
```
Comment (Camwyn) [inline src/resources/postcss/views/full/_single-event.pcss:262]: No judgement, but have to ask: Is there a good reason to vary this from existing sizes?
```
--font-size-2: 14px;
--font-size-3: 16px;
```

File: src/resources/postcss/views/skeleton/_single-event.pcss [hunk 2]
```diff
@@ -258,15 +283,26 @@
...
+			margin-bottom: var(--spacer-8);
+		}
 	}
-
-	&.secondary {
-		margin-top: 0;
-
+	
+	/* Tickets Block */
+	&.primary + .event-tickets + .event-tickets,
```
Comment (Camwyn) [inline src/resources/postcss/views/skeleton/_single-event.pcss:298]: can we add a @todo to move this to ET styles at some point? It doesn't really belong here.
RSVP styles above too.

File: src/resources/postcss/views/skeleton/_single-event.pcss [hunk 3]
```diff
@@ -382,8 +419,30 @@
 			margin-bottom: var(--spacer-3);
 			width: 100%;
 
+			/**
+			 * TODO: This is done specifically for IE11 and because we could not
+			 * touch templates. Use object-fit: cover; in the future if IE11
+			 * support gets dropped or update the HTML template if possible.
+			**/
```
Comment (Camwyn) [inline src/resources/postcss/views/skeleton/_single-event.pcss:426]: Soon hopefully!
https://make.wordpress.org/core/2021/03/04/discussion-dropping-support-for-ie11/

**Review by Camwyn — APPROVED** [review]
Couple questions but nothing that blocks this from moving forward.

File: src/resources/postcss/views/full/_single-event.pcss (hunk 1 above)
Comment (alliuca) [reply src/resources/postcss/views/full/_single-event.pcss:262]: asking Rachel, thanks

File: src/resources/postcss/views/skeleton/_single-event.pcss (hunk 2 above)
Comment (alliuca) [reply src/resources/postcss/views/skeleton/_single-event.pcss:298]: sure, will do

File: src/resources/postcss/views/skeleton/_single-event.pcss (hunk 3 above)
Comment (alliuca) [reply src/resources/postcss/views/skeleton/_single-event.pcss:426]: oh yeah 🙌  `object-fit: cover` ftw

File: src/resources/postcss/views/full/_single-event.pcss (hunk 1 above)
Comment (alliuca) [reply src/resources/postcss/views/full/_single-event.pcss:264]: we're going for 14px

**Pull request merged**